storage/container_storage_interface/persistent-storage-csi-azure.adoc

```diff
@@ -0,0 +1,34 @@
+[id="persistent-storage-csi-azure-disk"]
+= Azure Disk CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
+
+:FeatureName: Azure Disk Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to Azure Disk storage assets, {product-title} installs the Azure Disk CSI Driver Operator and the Azure Disk CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* The _Azure Disk CSI Driver Operator_ provides a storage class by default that you can use to create persistent volume claims (PVCs). You also have the option to create the Azure storage class as described in xref:../../storage/persistent_storage/persistent-storage-azure.adoc#persistent-storage-using-azure[Persistent storage using Azure].
```

Comment:
I would reword the second sentence here:

> You can create additional storage classes, as described in $LINK, to accommodate various storage profiles.

Reply:
The internal provisioner is used in the $LINK, how about adding some description here, for the CSI driver, we should use the `disk.csi.azure.com` provisioner?

In case the CSI driver has different parameters from in-tree plugin, maybe we can add more reference doc: https://github.com/openshift/azure-disk-csi-driver/blob/master/docs/driver-parameters.md?